exercise7.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def calculate_room_measurement_feet(length, width):
```

Comment:
I'd recommend naming this as `calculate_room_measurement`. You don't need to point out that it is in feet here. That may be better said in a docstring for this function.